engines/scumm/he/sound_he.cpp

```diff
@@ -780,7 +796,11 @@ void SoundHE::playHESound(int soundID, int heOffset, int heChannel, int heFlags,
 	}
 }
 
-void SoundHE::tryLoadSoundOverride(int soundID, Audio::RewindableAudioStream **stream) {
+Audio::RewindableAudioStream *SoundHE::maybeLoadSoundOverride(int soundID, int *duration) {
```

Comment:
tryLoadSoundOverride() sounds better as a name, IMHO. If we end up renaming the function, we should also rename the warnings that it may produce, since these are still prefixed as tryLoadSoundOverride